tests/upgrades/test_client.py

```diff
@@ -47,22 +45,7 @@
 
 
 @pytest.fixture(scope='module')
-def module_product(default_org):
-    return entities.Product(organization=default_org).create()
-
-
-@pytest.fixture(scope='module')
-def module_lce_library(default_org):
-    """Returns the Library lifecycle environment from chosen organization"""
-    return (
-        entities.LifecycleEnvironment()
-        .search(query={'search': f'name={ENVIRONMENT} and organization_id={default_org.id}'})[0]
-        .read()
-    )
-
-
-@pytest.fixture(scope='module')
-def sat6tools_repo(default_org):
+def sat6tools_repo(default_org, target_sat):
```

Comment:
There seems to be no use for this feature, can we remove it?

Reply:
Sure, we can. Updated.